src/dbnode/storage/fs.go

```diff
@@ -184,7 +184,6 @@ func (m *fileSystemManager) Run(
 }
 
 func (m *fileSystemManager) Report() {
-	m.databaseCleanupManager.Report()
```

Comment:
Hm, shouldn't the warm flush cleanup report still come from here?

Reply:
Ah yea I had previously removed it and then re-added it once I realized I needed to split up the cleanup logic.